flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/service/AbstractFlinkService.java

```diff
@@ -160,19 +163,31 @@ public KubernetesClient getKubernetesClient() {
         return kubernetesClient;
     }
 
+    public CuratorFrameworkWithUnhandledErrorListener getCurator(Configuration conf) {
```

Comment:
Wrapped ZK client creation in a separate method so that it's easier to mock once I get to unit tests